sdk/ai/ci.yml

```diff
@@ -50,10 +50,14 @@ extends:
     #    Selection: sparse
     #    GenerateVMJobs: true
     Artifacts:
-    - name: azure-ai-projects
-      safeName: azureaiprojects
-    - name: azure-ai-inference
-      safeName: azureaiinference
+    # Uncommentbefore merging to main branch
+    # - name: azure-ai-projects
+    #  safeName: azureaiprojects
+    #- name: azure-ai-inference
+    #  safeName: azureaiinference
+    #######################################
```

Comment:
Please uncomment the azure-ai-inference parts above. Leave the azure-ai-projects commented out.